README.adoc

```diff
@@ -1,22 +1,174 @@
-= Collaborative exams
-:sectanchors:
-
-image:https://travis-ci.org/badga/Collaborative-exams.svg?branch=master["Build Status", link="https://travis-ci.org/badga/Collaborative-exams"]
-
-A Java project developed during the https://github.com/oliviercailloux/java-course[Java course] taught at Université Paris-Dauphine (in http://www.mido.dauphine.fr/[MIDO] M2 MIAGE). The code is jointly copyrighted by Olivier Cailloux and the following students:
-
-* Jeff Azzam
-* Sid Ahmed Imloul
-* Sofiath Lanignan
-* Lamine Tamendjari
-* Omar Yaghi
-* Abdelmadjid Yous
-
-The code is hereby published under the MIT License, with their permission.
-
-== Original description
-Ce projet permettra à des enseignants de créer des ensembles de questions et réponses. Ces questions peuvent être réutilisées par d’autres enseignants et combinées pour créer des questionnaires ou des sondages. Les étudiants pourront se servir de ces ensembles de questions pour s’entrainer.
-
-=== Retrouvez notre application en suivant le lien ci dessous
-http://collaborative-exams.eu-gb.mybluemix.net/
-
+= Première itération 
+
+*Amine* 
+
+Question
+
+* Interfaces et objets pour Question et objets associés. Une question comprend un énoncé (« phrasing »), une langue, un ensemble de réponses possibles, chacune associée à l’information indiquant si cette réponse est correcte, un identifiant (int), et un auteur. Un auteur est identifié par un id (int) et a un e-mail. Utiliser un objet Person pour les auteurs. Un énoncé et une réponse sont des textes simples. L’ensemble de réponses peut être remplacé par une information « Y/N » ou « T/F » ou « free », indiquant une question oui / non, ou vrai / faux, ou de réponse libre. Créer à cet effet une interface MultipleChoiceQuestion qui étend Question. L’auteur peut associer un identifiant personnel (string) à une question. Ceci est stocké hors de l’objet Question. Cet identifiant peut changer. (La paire (auteur, identifiant) doit être unique.) Une question est immuable. (2)
+
+
+
+* Servlet NewQuestion(question): id, qui reçoit une question en format JSON ; servlet GetQuestion(id): question, qui envoie une question à partir de son id. (1)

```

Comment:
Ça fait trop pour une première itération. Faites seulement Question.

Reply:
(Ceci dit, rien ne vous empêche de prendre de l’avance sur le calendrier. Mais il est bon de marquer clairement les limites de chaque itération néanmoins.)